examples/projects/typescript-vanilla-with-rollup/README.md

```diff
@@ -6,7 +6,7 @@ To run locally:
 2. `npm run start`
 3. [localhost app](http://localhost:10001)
 
-You should see a BPMN diagram displayed in the page
+You should see a BPMN rendered in the page
```

Comment:
ℹ️ I would keep 'diagram' here for clarity.
The javascript project requires change as well.